api/circulation.py

```diff
@@ -414,56 +425,147 @@ def __repr__(self):
         )
 
 
-class BaseCirculationAPI:
-    """Encapsulates logic common to all circulation APIs."""
+class BaseCirculationAPISettings:
+    """Not an inheritable BaseSettings object, this just holds reusable settings"""
 
-    # Add to LIBRARY_SETTINGS if your circulation API is for a
-    # distributor which includes ebooks and allows clients to specify
-    # their own loan lengths.
-    EBOOK_LOAN_DURATION_SETTING = {
-        "key": Collection.EBOOK_LOAN_DURATION_KEY,
-        "label": _("Ebook Loan Duration (in Days)"),
-        "default": Collection.STANDARD_DEFAULT_LOAN_PERIOD,
-        "type": "number",
-        "description": _(
-            "When a patron uses SimplyE to borrow an ebook from this collection, SimplyE will ask for a loan that lasts this number of days. This must be equal to or less than the maximum loan duration negotiated with the distributor."
+    ebook_loan_duration: Optional[PositiveInt] = FormField(
```

Comment:
It looks like this is used in two different files: 
- axis.py 
- bibliotecha.py

But the type we use in each of those cases is different then what is specified on the base class. Instead of implementing it like this, maybe we could implement this as a couple real base classes, and use inheritance. That way there won't be the possibility of having different types on the subclasses, and we can use type checks to determine if a class has this setting or not.

Reply:
That makes sense :+1: